man/systemd-mount.xml

```diff
@@ -340,6 +340,23 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--canonicalize=</option></term>
+
+        <listitem>
+          <para>Controls whether the specified path shall be canonicalized on the client side before
+          requesting the operation or not. Takes a boolean parameter, defaults to true. Note that for
+          non-local operation (i.e. when <option>--machine=</option> or --<option>--host=</option> are used)
+          canonicalization is implicitly turned off.</para>
+
+          <para>Canonicalization of path entails resolving of symlinks and <literal>..</literal> path
+          elements. If canonicalization is disabled and the path contains a symlink element or
+          <literal>..</literal> the operation will fail.</para>
```

Comment:
Please also mention that fstab's `LABEL=...` style specification is refused when canonicalization is disabled.